.travis.yml

```diff
@@ -13,7 +12,7 @@ script:
 jobs:
   include:
   - stage: Lint and Test
-    script: npm run lint && gulp test:generateCIConfig && gulp test:visual
+    script: npm run lint && node ./ci-scripts/generateBackstopConfig.js && npm run test
```

Comment:
This additional script is needed as linux containers (Travis) cannot use `host.docker.internal` like local tests do

Reply:
Since the `generateBackstopConfig.js` file has `#!/usr/bin/env node` in it, I _think_ you can just invoke it like a shell script (without having to run `node`).  One caveat is the script file will need to be executable (`chmod 755`).

Reply:
Also, should this be wrapped into a script definition in `package.json`?  If for no other reason, just a way to run it isolated.

Reply:
yeah I went back and forth about that, I was trying to minimize package.json script bloat